examples/satellite-azure/README.md

```diff
@@ -50,19 +51,40 @@ terraform {
 
 ## Usage
 
+1. Create a copy of tfvars file.
+
+```
+cp inputs.tfvars.template inputs.tfvars
+```
+
+1. Edit it with your own variable definitions
```

Comment:
fix the numbering here between each step (1. 1. 1. 1.)

Reply:
Numbering fix